test/unit/module/test_api.py

```diff
@@ -66,3 +66,8 @@ def test_issues_template(self):
         filename = "test/fixtures/templates/bad/issues.yaml"
         matches = self.helper_lint_string_from_file(filename)
         self.assertEqual(["E1012"], [match.rule.id for match in matches])
+
+    def test_sam_template(self):
```

Comment:
This would fail prior to this change since the globals section is invalid in normal Cloudformation templates.